Xplat/src/main/java/vazkii/botania/common/block/flower/functional/LooniumBlockEntity.java

```diff
@@ -50,7 +53,6 @@
 public class LooniumBlockEntity extends FunctionalFlowerBlockEntity {
 	private static final int COST = 35000;
 	private static final int RANGE = 5;
-	private static final String TAG_LOOT_TABLE = "lootTable";
```

Comment:
Removing this is not ideal as it was added to allow mapmakers to override the loonium's loot table.
Unfortunately, all existing-placed flowers will have the dungeon loot table set here too.

How about we:
1. Rename the field to `customLootTable`, and default it to `null` for new flowers (don't write it to NBT in that case). Keep the tag key the same though for compatibility.
2. If nonnull, use this table, otherwise use the structure selection logic.
3. In the changelog (please add an entry in web/changelog.md), note that this change only applies to newly crafted looniums (old ones are going to keep their override even when broken because that's how it's supposed to be).

Reply:
Just to confirm - you want the nbt tag to be lootTable, not customLootTable, so that existing looniums with a set loot table retain it?

Reply:
In the currently released version of botania, if a loonium with a custom loot table set is broken, the dropped item doesn't have that NBT saved, and reverts to the normal pool if placed again - is that intentional?

Reply:
Huh, that's strange. Let me check.

Reply:
Ok, it doesn't, but that's a separate bug (the generated loot table needs logic to preserve the tag). As for your other question, yeah, let's keep the nbt tag as `lootTable`, but rename the code variable to express what it really means.